BaikalStandalone/Application/cl_render.cpp

```diff
@@ -790,6 +790,11 @@ namespace Baikal
             m_post_effect->SetParameter("visible_devices", settings.visible_devices);
             m_post_effect->SetParameter("start_spp", settings.denoiser_start_spp);
             break;
+        case DenoiserType::kRIF:
+            AddPostEffect(m_primary, PostEffectType::kRIFDenoiser);
```

Comment:
Maybe kMIOpenDenoiser -? RIF - is only interface of image library - it can use TF under the hood...